### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by gem-sakshiagrawal**

### CODE REVIEW

File: gempyp/engine/engine.py
[the remark this answers is not in the document]
Comment (gem-sakshiagrawal) [reply gempyp/engine/engine.py:1]: I had taken pull from beta release branch , then resolved conflicts and pushed it to remain consistent

[the remark this answers is not in the document]
Comment (gem-tanya-agarwal) [reply gempyp/engine/engine.py:1]: okay

File: gempyp/engine/engine.py [hunk 1]
```diff
@@ -846,11 +852,14 @@ def isDependencyPassed(self, testcase: Dict) -> bool:
             ","))) - set([""]))  # if testcase.get("DEPENDENCY", None) else listOfTestcases
         for dep in listOfTestcases:
 
-            dep_split = list(dep.split(":"))
+            dep_split = list(map(lambda i: i.strip(), list(dep.split(":"))))
```
Comment (gem-sachinchand) [inline gempyp/engine/engine.py:855]: try to remove this line as we don't need this. But before removing please test it throughly

Comment (gem-sakshiagrawal) [reply gempyp/engine/engine.py:855]: Without this, it is giving ERR. But after adding this line it is working as expected.

**Pull request merged**